docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,83 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
+
+To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters.
```

Comment:
Do we have anywhere we could link them to help understand what intrinsic parameters are? Maybe the camera page?

Reply:
Tried to address with 2 suggestioins below-- lmk what you think! @npentrel